Terminal.Gui/Views/TextView.cs

```diff
@@ -1962,7 +1962,25 @@ protected virtual void ColorNormal (List<Rune> line, int idx)
 		/// <param name="idx"></param>
 		protected virtual void ColorSelection (List<Rune> line, int idx)
 		{
-			Driver.SetAttribute (ColorScheme.Focus);
+			Driver.SetAttribute (new Attribute (ColorScheme.Focus.Background, ColorScheme.Focus.Foreground));
+		}
+
+		/// <summary>
+		/// Sets the <see cref="View.Driver"/> to an appropriate color for rendering the given <paramref name="idx"/> of the
+		/// current <paramref name="line"/>.  Override to provide custom coloring by calling <see cref="ConsoleDriver.SetAttribute(Attribute)"/>
+		/// Defaults to <see cref="ColorScheme.Focus"/>.
+		/// </summary>
+		/// <param name="line"></param>
+		/// <param name="idx"></param>
+		protected virtual void ColorReadOnly (List<Rune> line, int idx)
```

Comment:
Should this be named `SetReadOnlyColor` instead?

Maybe since they are all currently `protected` we use this PR to fix all these methods to use correct naming?

Reply:
I agree. I will change them.

Reply:
Done.